src/preload.py

```diff
@@ -389,6 +377,38 @@ def list_images(image):
     return _list_images(image)
 
 
+def get_docker_init_file_content(image):
+    with mount_context_manager(image, 2) as mountpoint:
+        path = os.path.join(
+            mountpoint,
+            'lib',
+            'systemd',
+            'system',
+            'docker.service',
```

Comment:
Presumably this works for ResinOS, but it's not guaranteed to work for images more generally. Docker's recommended location for setting like this is actually `/etc/docker/daemon.json`, and anybody starting docker outside systemd could be passing in these args absolutely elsewhere.

Is that a concern? What about with host apps in the future? Is there any more reliable way we could work out the filesystem that's being used?

Looking at some of my devices, I've got a `/var/lib/docker/aufs` folder on new ResinOS 2 devices, and I've got `/var/lib/docker/overlay2` on my local machine (which uses overlay2). Is checking for the existence of those practical, and any more reliable?